npm-shrinkwrap.json

```diff
@@ -1,70 +1,72 @@
 {
   "name": "perseo",
-  "version": "0.4.0-next",
```

Comment:
I understand this is a kind of dependencies file... There are many changes, even in packages not related with Twitter. Is this ok?